### PR TITLE
79 create complaint

### PR DESCRIPTION
**Pull request by Klemensas**
Part of https://github.com/spry-group/trashradar-api/issues/79

Used a static entity list for now.
Not sure what should be sent for the `tweet_status` field, left it out for now.

### CODE REVIEW

**Comment by carlosabbate** [comment]
The form is failing on summit with error `Failed to load resource: the server responded with a status of 400 (Bad Request)`.

**Comment by Klemensas** [comment]
Yeah currently there are no entities so I've just added some mock entities, but actually creating a complaint will not work until we have some entities with the endpoint as is.

File: src/app/main/complaint/complaint.component.ts [hunk 1]
```diff
@@ -28,18 +31,35 @@ export class ComplaintComponent implements OnInit {
...
     private mapsAPILoader: MapsAPILoader,
     private ngZone: NgZone,
     private store: Store<any>,
     private complaintActions: ComplaintActions,
-  ) {}
+  ) {
+    this.complaintStateSubscription = this.store.select(getComplaintState)
+                                      .subscribe(({ ids, entities, selectedComplaintId, error }) => {
+      if ((error === null || error === undefined) && selectedComplaintId) {
+        this.successMessage = 'Complaint created succesfully!'
```
Comment (marton987) [inline src/app/main/complaint/complaint.component.ts]: `;` missing

**Comment by marton987** [comment]
LGTM

**Pull request merged**